client/elements/text/sc-text-image.html

```diff
@@ -65,6 +62,14 @@
                   },
                   lastDetail: {
                       type: Object
+                  },
+                  showUrl: {
+                      type: String,
+                      computed: '_getShowUrl(imageData)'
```

Comment:
(not important): `_getShowUrl(imageData)` could be inlined in the template

Reply:
doesn't work in this case because the viewer imports src and it tends to see the function as a sting rather than execute the function. So your url becomes `https://suttacentral.net/[[_getShowUrl(imageData)]]` and it gives an error.